src/HttpClientInterception/InterceptingHttpMessageHandler.cs

```diff
@@ -41,6 +41,12 @@ public InterceptingHttpMessageHandler(HttpClientInterceptorOptions options, Http
         _options = options ?? throw new ArgumentNullException(nameof(options));
     }
 
+#if NET6_0_OR_GREATER
+    /// <inheritdoc />
+    protected override HttpResponseMessage Send(HttpRequestMessage request, CancellationToken cancellationToken) =>
+        SendAsync(request, cancellationToken).ConfigureAwait(false).GetAwaiter().GetResult();
```

Comment:
Yeah this is a bit ick, but as it's a test library and probably quite a minority subset of cases (as it's taken 2 years to hit this) this is fine.